databases/mysql57/Portfile

```
@@ -8,8 +8,8 @@ set name_mysql      ${name}
 version             5.7.42
 set boost_version   1.59.0
 # Set revision_client and revision_server to 0 on version bump.
-set revision_client 0
-set revision_server 0
+set revision_client 1
+set revision_server 1
```

The server subport just installs the launchd plist and creates some directories so it doesn't need a revbump.
```suggestion
set revision_server 0
```